.eslintrc.json

```diff
@@ -1,151 +1,109 @@
 {
```

Comment:
@olajide1234 why do we keep tinkering with this file in almost every PR?

Reply:
The changes were majorly formatting. I will watch out to ensure I don't reformat it in subsequent PRs and will also inform the team.